_includes/welcome.md

```diff
@@ -1,54 +1,60 @@
 # Bem vindo
 
-Existe muita informação obsoleta na Web que desnorteia novos programadores PHP,
-espalhando más práticas e códigos inseguros. _PHP: Do Jeito Certo_ é uma referência
-rápida e fácil de ler, introduzindo desenvolvedores às melhores práticas, renomados
-padrões de código e links para tutoriais competentes pela Web.
-
-É importante entender que _não existe uma maneira canônica de usar PHP_. Este site introduz novos desenvolvedores PHP 
-às melhores práticas, opções
-disponíveis e boas informações, que por muitas vezes, quando descobertas, já é tarde
-demais. Além disso leva o desenvolvedor a ter conceitos maduros sobre coisas que eram
-feitas durante anos e nunca foram reconsideradas. Esse site não é para dizer quais
-ferramentas você deve utilizar, mas ao invés disso, oferecer sugestões e múltiplas
-opções, e quando possível explicar as diferenças de acordo com casos de uso.
-
-Este é um documento "vivo", que continuará sendo atualizado com mais informações úteis
-e exemplos, assim que eles estiverem disponíveis.
+Há muitas informações desatualizadas na Web que levam os novos usuários de PHP a se desorientar, propagando más práticas
+e código inseguro. _PHP: Do Jeito Certo_ é uma referência rápida e fácil de ler para os padrões de código populares do
+PHP, links para tutoriais confiáveis ao redor da Web e o que os contribuidores consideram como melhores práticas no
+momento.
+
+_Não há uma maneira canônica de usar PHP_. Este site visa introduzir novos desenvolvedores PHP a alguns tópicos que eles
```

Comment:
Acho que trocar `canônica` por `única` torna o texto mais fácil de digerir. Pra deixar mais natural em pt-br eu tomaria a liberdade de inverter a negação dizendo que `há várias maneiras de usar PHP`.

Também recomendo trocar `novos desenvolvedores` por `iniciantes` em favor de adotar uma linguagem neutra em gênero. Isso vai te forçar a repensar o resto da frase. Vou deixar aqui minha sugestão:

> _Há várias maneiras de usar PHP_. Este site visa introduzir iniciantes em PHP a alguns tópicos que talvez só seriam descobertos tarde demais, e pretende oferecer a profissionais experientes algumas ideias sobre tópicos que vêm sendo utilizados por anos sem questionamento. Este website [..]